prober/tcp.go

```diff
@@ -98,6 +98,13 @@ func ProbeTCP(ctx context.Context, target string, module config.Module, registry
 		Name: "probe_ssl_last_chain_expiry_timestamp_seconds",
 		Help: "Returns last SSL chain expiry in unixtime",
 	})
+	probeSSLLastInformation := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "probe_ssl_last_chain_info",
+			Help: "Contains SSL leaf certificate information",
+		},
+		[]string{"sha256"},
```

Comment:
sha256 is a bit vague

Reply:
Going to change it to `fingerprint_sha256` or do you got something better in mind?

Reply:
That sounds good to me.

Reply:
Changed it in [92dfa8f](https://github.com/prometheus/blackbox_exporter/pull/678/commits/92dfa8f4d4eaa114ab26241a5301f9a1cff7d6af)